tools/camera_combinexsannos/abims_CAMERA_combinexsAnnos.xml

```diff
@@ -1,57 +1,51 @@
-<tool id="abims_CAMERA_combinexsAnnos" name="CAMERA.combinexsAnnos" version="2.2.6+camera@TOOL_VERSION@-galaxy0">
-
-    <description>Wrapper function for the combinexsAnnos CAMERA function. Returns a dataframe with recalculated annotations.</description>
-
+<tool id="abims_CAMERA_combinexsAnnos" name="CAMERA.combinexsAnnos" version="2.2.6+camera@TOOL_VERSION@+galaxy1" profile="20.09">
+    <description>Wrapper function for the combinexsAnnos CAMERA function. Returns a dataframe with recalculated annotations</description>
     <macros>
         <import>macros.xml</import>
     </macros>
-
     <expand macro="requirements"/>
     <expand macro="stdio"/>
-
     <command><![CDATA[
-        @COMMAND_RSCRIPT@/CAMERA_combinexsAnnos.r
+        LC_ALL=C Rscript '$__tool_directory__/CAMERA_combinexsAnnos.r'
         image_pos '$image_pos'
         image_neg '$image_neg'
 
-        pos $pos tol
-        $tol ruleset
-        $ruleset keep_meta
-        $keep_meta
+        pos $pos
+        tol $tol
+        ruleset '$ruleset'
+        keep_meta $keep_meta
 
         convertRTMinute $export.convertRTMinute
         numDigitsMZ $export.numDigitsMZ
         numDigitsRT $export.numDigitsRT
 
-    ]]></command>
-
+    ]]>    </command>
+    <required_files>
+        <include path="lib.r" />
```

Comment:
This is needed, because Galaxy does not know that lib.r belongs to the tool. `CAMERA_combinexsAnnos.r` can be deduced from the command section, but `lib.r` not. This feature is new and needs the 21.09 Galaxy release. 
The advantage is that it will make this tool for in pulsar.

Reply:
Good to know!